contrib/dracut/90zfs/zfs-load-key.sh.in

```diff
@@ -38,6 +38,9 @@ if [ "$(zpool list -H -o feature@encryption $(echo "${BOOTFS}" | awk -F\/ '{prin
     # if the root dataset has encryption enabled
     ENCRYPTIONROOT=$(zfs get -H -o value encryptionroot "${BOOTFS}")
     if ! [ "${ENCRYPTIONROOT}" = "-" ]; then
+	KEYSTATUS="$(zfs get -H -o value keystatus "${ENCRYPTIONROOT}")"
```

Comment:
This file uses spaces. Can you change these from tabs to 8 spaces (in this file only)?